adapter/config/ingress/controller.go

```diff
@@ -56,15 +56,16 @@ func NewController(client kubernetes.Interface, mesh *proxyconfig.ProxyMeshConfi
 	// queue requires a time duration for a retry delay after a handler error
 	queue := kube.NewQueue(1 * time.Second)
 
-	glog.V(2).Infof("Ingress controller watching namespaces %s", options.AppNamespace)
+	glog.V(2).Infof("Ingress controller running in namespace %s, watching namespaces %s",
+		options.Namespace, options.WatchedNamespace)
 	// informer framework from Kubernetes
 	informer := cache.NewSharedIndexInformer(
 		&cache.ListWatch{
 			ListFunc: func(opts meta_v1.ListOptions) (runtime.Object, error) {
-				return client.ExtensionsV1beta1().Ingresses(options.AppNamespace).List(opts)
+				return client.ExtensionsV1beta1().Ingresses(options.WatchedNamespace).List(opts)
```

Comment:
What if I want one ingress controller for all namespaces? Do I have an option for that ? That is the default and common installation.

Reply:
`WatchedNamespace = metav1.NamespaceAll`?

Reply:
In that case you do not pass the argument -a when starting the controller, and it will by default watch all Namespaces. I explicity added the Namespace all in -a arg, I hope this clarifies.

discoveryCmd.PersistentFlags().StringVarP(&flags.controllerOptions.WatchedNamespace, "app namespace",
		"a", metav1.NamespaceAll,
		"Restrict the applications namespace the controller manages; if not set, controller watches all namespaces")

Reply:
Yep. Thanks